pkg/ui/workspaces/cluster-ui/src/statementsPage/statementsPageRoot.tsx

```diff
@@ -41,7 +41,7 @@ export const StatementsPageRoot = ({
           {`A statement fingerprint represents one or more completed SQL
           statements by replacing literal values (e.g., numbers and strings)
           with underscores (_).\nThis can help you quickly identify
-          frequently executed SQL statements and their latencies.`}
+          frequently executed SQL statements and their latencies. `}
```

Comment:
Is the trailing space intended?

Reply:
Yes, that was one of the issues, because this is followed by a learn more link, and was wrong without this space between them

Reply:
Ah I see